fanboy-anticomments.txt

```diff
@@ -17,30 +17,50 @@
 !
 ! Generic element
 !
+###commentlistrecent
 ###comments-wrapper-coral
 ###disqus_thread
 ##.asset-comments
 ##.c-comments
 ##.cf.comments-content
+##.commentcontainer
 ##.comments-container
+##.commentsList
 ##.disqus.post-widget
 ##.fb_iframe_widget.fb-comments
 ##.js_comments-iframe
+##.o-comments-link
+##.ui-comments
 ##.vf-comments
 ##.vf-widget
 ##.widget_disquswidget
 ##.xenforo-comment-widget
+##article-facebook-comments
 ! First-party elements
 !
 dailymail.co.uk##.article-reader-comments
-techpowerup.com##.comments
-youtube.com###comments
+techpowerup.com,youtube.com,vg.no,invidio.us,ghacks.net,nintendolife.com,thejournal.ie,e621.net###comments
+presstv.com##.item-comments
+nintendolife.com###reply
+m.youtube.com##ytm-comment-section-renderer
+color-hex.com##a[href="#comments"]
+rule34.xxx##div[id^=c][style="display:inline;"]
+theregister.co.uk##.story_img_wrap > .comments > .count
+||comments.ign.com^
+||ws-comments.ign.com^
```

Comment:
Why not just make a `disqus.com` and `disquscdn.com` rule? that would hit much broader and stop a lot of tracking Privacy data collecting at ones.

`disqus.com`: <https://www.mypdns.org/T1042>
`disquscdn.com`: <https://www.mypdns.org/T1043>

Reply:
The list seems to already contain `||disqus.com^$third-party`.

I presume that adding an entry against `||disquscdn.com` in this particular list, would've depended on whether it's only used in comment sections or not.

Reply:
You are right on the `disquscdn.com` 

I have just only seen it on the `disqus.com` and with comment integration, but it is of curse also completely blocked in my DNS, so I never see it in other situations, then like this.